pages/1.13/metrics/index.md

```diff
@@ -24,6 +24,9 @@ Telegraf is included in the DC/OS distribution and runs on every host in the clu
 
 For more information about the list of metrics that are automatically collected by DC/OS, read [Metrics Reference](/1.13/metrics/reference/) documentation.
 
+## Monitoring and Visualization
+We recommend installing the DC/OS Monitoring Service to monitor and visualize metrics in your DC/OS cluster. See the [service documentation](/services/dcos-monitoring/) for instructions on how to install and use the service.
```

Comment:
#4 - we are planning on releasing GA dcos-monitoring with dcos 1.13, so the docs would go under dcos-monitoring instead of beta-dcos-monitoring. That route doesn't exist yet because we haven't yet released it. When do 1.13 docs go live? I'll have it point to the beta route now, and we'll need to remember to change them once we release GA dcos-monitoring.